routing/router_test.go

```diff
@@ -146,6 +146,82 @@ func TestFindRoutesFeeSorting(t *testing.T) {
 	}
 }
 
+// TestFindRoutesFeeCutoff asserts that the FindRoutes method returns no
+// routes with fees greater than the maxFee cutoff.
+func TestFindRoutesFeeCutoff(t *testing.T) {
+	t.Parallel()
+
+	const startingBlockHeight = 101
+	var maxFee lnwire.MilliSatoshi
+	ctx, cleanUp, err := createTestCtx(startingBlockHeight, basicGraphFilePath)
+	defer cleanUp()
+	if err != nil {
+		t.Fatalf("unable to create router: %v", err)
+	}
+
+	paymentAmt := lnwire.NewMSatFromSatoshis(100)
+	target := ctx.aliases["luoji"]
+
+	// There are two possible paths from roasbeef to luo ji, with fees of
```

Comment:
Missing a period at the end of sentences in a few comments within this test case (won't comment on them all in order to reduce review comment clutter).